pokerole_irl/user_app/models.py

```diff
@@ -11,3 +11,25 @@ class Profile(models.Model):
 
     def __str__(self):
         return self.user.username
+
+
+class Setting(models.Model):
+    owner = models.ForeignKey(User, on_delete=models.CASCADE)
+    members = models.ManyToManyField(User, related_name='+', blank=True)
+    name = models.CharField(max_length=30)
+    description = models.TextField(max_length=1000, default="")
+
+
+    def __str__(self):
+        return self.name
+
+class Character(models.Model):
+    owner = models.ForeignKey(User, on_delete=models.CASCADE)
+    setting = models.ForeignKey(Setting, on_delete=models.CASCADE)
```

Comment:
I don't like the CASCADE here. The setting is not necessarily owned by the same user as the character, so someone else deleting their setting would also delete this other user's character.

For the same reason, this field should be nullable (blank=True, null=True)